README.md

```diff
@@ -5,30 +5,74 @@ This is the repository of the referee box for the RoCKIn competition (http://roc
 
 
 ## Installation
-To install the RoCKIn referee box, please install the prerequisites of the LLSF referee box which are described here: https://trac.fawkesrobotics.org/wiki/LLSFRefBox/Install
+The RoCKIn Central Factory Hub (CFH) can be installed on most linux distributions. However, some require dependencies being built from source. The current recommended setup is Ubuntu 14.04, with Boost 1.54. We support and test on Ubuntu 14.04 LTS. For other non-supported distributions and setups see Alternative Installation Hints at the end of this readme.
 
-Additionally, the following debian packages need to be installed:
+### Officially Supported Setup: Ubuntu 14.04, Boost 1.54
 
-    sudo apt-get install libssl-dev libelf-dev mongodb-dev mongodb-clients mongodb libzmq3-dev
-  
-## Compiling the RoCKIn@Work Referee Box
-### Cloning the Git repository:
+1. The CFH is based off LLSF refbox, and uses [clips](http://clipsrules.sourceforge.net/). Add [Tim's PPA](https://launchpad.net/~timn/+archive/ubuntu/clips) for clips:
```

Comment:
1. LLSF is already mentioned in the beginning, no need to repeat here.
2. No need to explicitly mention the use of CLIPS (else which dependencies to enumerate, which not?)
3. Tim -> Tim Niemueller